provider-ci/internal/pkg/templates/bridged-provider/Makefile

```
@@ -167,5 +171,17 @@ ci-mgmt: .ci-mgmt.yaml
 		--template bridged-provider \
 		--config $<
 
+.pulumi/bin/pulumi: HOME := $(WORKING_DIR)
+.pulumi/bin/pulumi: .pulumi/version
+	curl -fsSL https://get.pulumi.com | sh -s -- --version $(cat .pulumi/version)
+
+# Compute the version of Pulumi to use by inspecting the Go dependencies of the provider.
+.pulumi/version:
```

We want this to change when we update our `go.mod`.
```suggestion
.pulumi/version: provider/go.mod
```

---

Yeah, no, this breaks the shellcheck linter because it's shell-checking in a context where there is no provider/go.mod... I got a little burned out trying to work around this.

---

We use shellcheck to prevent us making mistakes. We shouldn't allow it to encourage bad practice. Instead of working around shellcheck, can we just disable it checking that line: https://www.shellcheck.net/wiki/Ignore.

---

https://github.com/pulumi/ci-mgmt/issues/660 that does not really work.